css/css-animations/parsing/keyframes-name-invalid.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <title>CSS Animations: parsing @keyframes name with invalid values</title>
+    <link rel="author" title="yisibl(一丝)" href="https://github.com/yisibl"/>
+    <link rel="help" href="https://drafts.csswg.org/css-animations/#typedef-keyframes-name">
+    <meta name="assert" content="@keyframes name supports the full grammar '<custom-ident> | <string>'.">
+    <script src="/resources/testharness.js"></script>
+    <script src="/resources/testharnessreport.js"></script>
+    <script src="/css/support/parsing-testcommon.js"></script>
+</head>
+<body>
+    <div>
+        <main id="main"></main>
+    </div>
+    <script>
+        function cleanup_main() {
+            while (main.firstChild)
+                main.firstChild.remove();
+        }
+
+        function set_style(text) {
+            let style = document.createElement('style');
+            style.innerText = text;
+            main.append(style);
+            return style;
+        }
+
+        function test_keyframes_name_invalid(keyframes_name) {
```

Comment:
It would probably be good for these two functions to share most of their code, so that it's clear they do almost the same thing.

Reply:
Thanks for your help with the review, it's now extracted to `/css/support/parsing-testcommon.js`.